e2e/tests/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
We have two tsconfig files, one in pages and one in tests. They are practically identical (only difference is noImplicitAny). We could move the tsconfig file directly under the e2e folder and use the same one for both. In the merged one check to see if we can set noImplicyAny to `true` which is preferred but if not then set to `false`.